site/utils/index.ts

```diff
@@ -19,3 +20,16 @@ export const redirectTo = (res: NextApiResponse | ServerResponse, location: stri
 
 export const getCsrfToken = (ctx: NextPageContext | NextPageContext): string =>
     (ctx.res as ResponseWithLocals)?.locals?.csrfToken ?? "";
+
+export const convertDateTimeToFormat = (dateOrTime: string, format: string) => dayjs(dateOrTime).format(format);
+
+export const formatTime = (time: string) => (time ? time.slice(0, -2) + ":" + time.slice(-2) : "");
```

Comment:
Do we have unit tests for these functions? It's hard to see what this one does

Reply:
I will write some

Reply:
done